### PR TITLE
fix: bump consola dep to 3.x to align with other nuxt 2 packages

### PR DESCRIPTION
**Pull request by rchl**
Addressing the https://github.com/nuxt/nuxt/issues/24343 issue which is about outdated `consola` being used in a couple of Nuxt 2 packages.

### CODE REVIEW

**Comment by rchl** [comment]
Bumped node requirement from 8.0.0 to 14.18.0 (required by consola 3.x). I don't expect this to be a breaking change for anyone these days. (If it is then they should blame themselves.)

File: src/reporters/consola.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const { colors, clearConsole } = require('../utils/log')
-const consola = require('consola')
+const { consola } = require('consola')
```
Comment (pi0) [inline src/reporters/consola.js]: This is an unnecessary change btw. Having default export is more backward compatible.

Comment (rchl) [reply src/reporters/consola.js]: Yeah, but you (Nuxt team) recommends the switch: https://github.com/unjs/consola/releases/tag/v3.0.0

Also, Consola 3.x is in dependencies so it's guaranteed to be used by this code.

Comment (pi0) [reply src/reporters/consola.js]: I guess I am aware 😆 Yes, still a default export for old dependencies like this worth since older consola version can be hoisted. npm ecosystem is funny...

Comment (rchl) [reply src/reporters/consola.js]: I think hoisting would only potentially be an issue for a code running from webpack context. This should probably only run at build time so I would think should be guaranteed to run its own unhoisted dependency (assuming that it's different from the hoisted one).

Unless someone uses yarn `resolutions` then all bets are off but I think that's users choice then.

In any case, do you want me to change it?
Note that this is one of many dependencies with Consola so it's not likely that all will consistently follow the same way.

Comment (pi0) [reply src/reporters/consola.js]: Yes please changing to default export would be nice. And yes this is almost the only dependency I am asking to change considering its age. Hoisting can still happen within `node_modules`.

**Comment by pi0** [comment]
@danielroe this looks good to me. feel free to merge and release when ready 🙏🏼

**Review by pi0 — APPROVED** [review]
